### PR TITLE
MAINT-46589 : Add Mention Processor and Html sanitize processor to the MESSAGE template param used in the activity with attached file

### PR DESCRIPTION
**Pull request by hakermi**
This PR should fix user mention in the activities with attached files and the activities with attached files + link preview and the display of the emojis in activities description

### CODE REVIEW

File: component/core/src/main/java/org/exoplatform/social/core/processor/MentionsProcessor.java [hunk 1]
```diff
@@ -41,6 +42,8 @@
 public class MentionsProcessor extends BaseActivityProcessorPlugin {
 
   private static final Pattern pattern = Pattern.compile("@([^\\s]+)|@([^\\s]+)$");
+  private static final String MESSAGE_PARAM = "MESSAGE";
```
Comment (ahamdi) [inline component/core/src/main/java/org/exoplatform/social/core/processor/MentionsProcessor.java]: COuld you move this variable to a common place, check if it is not already available

**Review by boubaker — DISMISSED** [review]
It does work now on new AS, knowing that the old AS will disappear for 6.2, I think this should be committed and tested for 6.1.x only. WDYT ?

**Comment by hakermi** [comment]
> It does work now on new AS, knowing that the old AS will disappear for 6.2, I think this should be committed and tested for 6.1.x only. WDYT ?

Agreed.
fyi @mzorai 
cc @ahamdi WDYT?

**Comment by ahamdi** [comment]
@boubaker Yes that was what we agreed on Maintenance team 2 days ago. @hakermi please update the PR for 6.1

**Comment by hakermi** [comment]
> @boubaker Yes that was what we agreed on Maintenance team 2 days ago. @hakermi please update the PR for 6.1

Done

**Review by ahamdi — APPROVED** [review]

File: component/core/src/main/java/org/exoplatform/social/core/processor/MentionsProcessor.java (hunk 1 above)
Comment (hakermi) [reply component/core/src/main/java/org/exoplatform/social/core/processor/MentionsProcessor.java]: done

**Pull request merged**